README.rst

```diff
@@ -8,14 +8,14 @@ v2.0 API. This project is hosted on `Github
 Installation
 ============
 
-This library is released to PyPI, so the easiest way to install it is to use
-easy_install::
+This library is released to PyPI - the Python Package Index, so the easiest way to install it is to use
+pip::
```

Comment:
Nitpick - this change is incidental to the changes under review; I would have reserved this change for a separate PR. It can remain here, though.